server/routers/admin_dashboard.py

```
@@ -44,66 +44,82 @@ async def get_admin_stats(
-        recent_activities = db.execute(recent_activities_sql.bindparams(since_param=twenty_four_hours_ago)).scalar() or 0
+        # Total users count
+        total_users = db.query(func.count(User.id)).scalar() or 0
+        
+        # Active admins count
+        active_admins = db.query(func.count(User.id)).filter(User.is_admin.is_(True)).scalar() or 0
+        
+        # System health checks
+        total_crises = 0
+        try:
+            total_crises = db.query(func.count(Disaster.id)).filter(Disaster.archived.is_(False)).scalar() or 0
+        except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)